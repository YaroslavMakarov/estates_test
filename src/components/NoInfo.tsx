import React from 'react';
import { Container } from 'react-bootstrap';
import 'bootstrap/dist/css/bootstrap.min.css';

import { NoInfoContent } from '../styled/NoInfoStyles';

const NoInfo = () => (
        <Container className="d-flex align-items-center justify-content-center">
            <NoInfoContent>
                Sorry, don't have any information about this estate.
            </NoInfoContent>
        </Container>
);

export default NoInfo;