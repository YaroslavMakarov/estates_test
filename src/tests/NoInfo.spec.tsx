import React from 'react';
import { shallow } from 'enzyme';
import { Container } from 'react-bootstrap';

import NoInfo from '../components/NoInfo';

describe('NoInfio component', () => {
    const wrapper = shallow(<NoInfo />).find(Container);
    it('get NoInfo snapShot', () => {
        expect(wrapper.debug()).toMatchSnapshot();
    });  
});