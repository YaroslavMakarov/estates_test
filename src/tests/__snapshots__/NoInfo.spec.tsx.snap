// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoInfio component get NoInfo snapShot 1`] = `
"<Container className=\\"d-flex align-items-center justify-content-center\\" fluid={false}>
  <styled.div>
    Sorry, don&#39;t have any information about this estate.
  </styled.div>
</Container>"
`;
